src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,2 @@
+junit.jupiter.execution.parallel.enabled=true
+junit.jupiter.execution.parallel.mode.default=concurrent
```

Comment:
Is JUnit 5 involved?

Reply:
There are 3 potential things here

1) Parallel Modules
2) Forking Multiple JVMs per module test run and dividing Test Cases between them
3) Concurrency within a single VM - https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-parallel-execution

Junit 5 runner is relevant for the 3rd, but shouldn't affect the test classes, just a different test runner.  I may not have unlocked this yet...

Reply:
I'm not in a rush until I confirm all 3 are active and see if results are ok.